pkg/registration/supply_chain_factory.go

```diff
@@ -136,11 +137,14 @@ func (f *SupplyChainFactory) buildNodeMergedEntityMetadata() (*proto.MergedEntit
 		builder.PropertyCapacity: {},
 	}
 	fieldsUsedCapacityPeak := map[string][]string{
-		builder.PropertyUsed:     {},
-		builder.PropertyCapacity: {},
-		builder.PropertyPeak:     {},
+		builder.PropertyUsed:      {},
+		builder.PropertyCapacity:  {},
+		builder.PropertyPeak:      {},
+		builder.PropertyResizable: {},
 	}
 	mergedEntityMetadataBuilder := builder.NewMergedEntityMetadataBuilder()
+
+	mergedEntityMetadataBuilder.PatchField(ActionEligibilityField, []string{})
```

Comment:
Curious: why are we patching it as an empty list?

Reply:
We are patching the action eligibility section in the DTO. The 2nd argument is the path to the section, it is empty  because the section is at the top level in the entity dto.